functors task (1)/student/src/impl.cpp

```diff
@@ -1,2 +1,11 @@
 #include "impl.h"
 
+BeerBrand BeerOrganizer::operator()()
+{
+    if (currentBeerIndex == static_cast<int>(BeerBrand::Max))
+    {
+        currentBeerIndex = 1;
```

Comment:
_nice, but still magic counters in code_

Reply:
ok, changed to enum cast